code/parachain/runtime/composable/src/gates.rs

```diff
@@ -0,0 +1,114 @@
+use crate::{
+	prelude::*, weights, Balances, ReleaseCommittee, Runtime, RuntimeCall, RuntimeEvent,
+	RuntimeOrigin,
+};
+use common::{
+	fees::NativeExistentialDeposit, governance::native::ReleaseCollective, AccountId,
+	MaxStringSize, HOURS,
+};
+use composable_traits::account_proxy::ProxyType;
+use cumulus_primitives_core::relay_chain::BlakeTwo256;
+use frame_support::traits::{Contains, EitherOfDiverse, InstanceFilter};
+use frame_system::EnsureRoot;
+
+impl InstanceFilter<RuntimeCall> for ProxyType {
+	fn filter(&self, c: &RuntimeCall) -> bool {
+		match self {
+			ProxyType::Any => true,
+			ProxyType::Governance => matches!(
+				c,
+				RuntimeCall::Democracy(..) |
+					RuntimeCall::Council(..) |
+					//RuntimeCall::TechnicalCommittee(..) |
+					RuntimeCall::Treasury(..) |
+					RuntimeCall::Utility(..)
+			),
+			ProxyType::CancelProxy => {
+				matches!(c, RuntimeCall::Proxy(pallet_proxy::Call::reject_announcement { .. }))
+			},
+			ProxyType::Bridge => matches!(
+				c,
+				//RuntimeCall::Ibc(..) |
+				RuntimeCall::CumulusXcm(..) |
+					RuntimeCall::DmpQueue(..) |
+					RuntimeCall::UnknownTokens(..) |
+					RuntimeCall::XcmpQueue(..) |
+					RuntimeCall::RelayerXcm(..)
+			),
+		}
+	}
+	fn is_superset(&self, o: &Self) -> bool {
+		match (self, o) {
+			(x, y) if x == y => true,
+			(ProxyType::Any, _) => true,
+			(_, ProxyType::Any) => false,
```

Comment:
```suggestion
```

Reply:
it is same in picasso. i did not changed semantics copying code from there. so will not.fix in this pr even if it is not right.

Reply:
so imho we should recheck these value. so likely will do in other pr. for now that is 1 to 1 with picasso.